cmake/template/startup_robotology_superbuild.m.in

```diff
@@ -1,13 +1,13 @@
 %% startup_robotology_superbuild.m
 %
 %  Run this script only once to permanently add the required folders for using
-%  MATLAB toolbox (e.g. WBToolbox, whole-body-controllers) to your MATLAB path. 
+%  MATLAB toolbox (e.g. WBToolbox, whole-body-controllers, matlab-whole-body-simulator) to your MATLAB path.
 
 fprintf('\nMATLAB Toolbox\n');
 
 installDir   = '@YCM_EP_INSTALL_DIR@';
 mexDir       = [installDir, filesep, 'mex'];
-simLibDir    = [installDir, filesep, 'mex/+wbc/simulink'];
+simLibDir    = [installDir, filesep, 'mex/+wbc/simulink', ':../src/matlab-whole-body-simulator/lib/RobotDynamicsWithContacts'];
```

Comment:
@traversaro , actually I would like to remove this line altogether, because:
- The relevant files are no more in `+wbc/simulink` as we are all aware. This generates a warning in MATLAB.
- We don't need to add the `+wbc` path, as this folder is a package, and all the functions there are accessed through the namespace prefix `wbc.`.
- Considering the previous two points, `mexDir` path is enough, or we can set (for the sake of generality) `simLibDir = mexDir`.

CC @gabrielenava

Reply:
I would then add a `+wbs` (whole-body-simulator) where I would copy all the files required for the `matlab-whole-body-simulator` installation.